docs/CONTRIBUTING.rst

```diff
@@ -28,7 +28,7 @@ experience and diverse personal backgrounds.
 User feedback
 -------------
 
-PUDL's goal is to help people use data to make change in the US energy landscape.
+PUDL's goal is to help people use data to make changes in the US energy landscape.
```

Comment:
Here we are talking about the un-countable, abstract notion of "making change" instead of specific changes being made, so `make change` is correct.

Reply:
Thanks @jdangerx for the great info! 
I have fixed the requested changes.